azure-pipelines.yml

```diff
@@ -73,34 +73,35 @@ jobs:
       continueOnError: true
       condition: not(succeeded())
 
-- job: Windows_Spanish
+- job: Linux_Spanish
   pool:
-   name: NetCorePublic-Pool
-   queue: buildpool.windows.10.amd64.es.vs2017.open
+    vmImage: 'ubuntu-latest'
   variables:
-    _os: Windows
+    _os: Linux
+    LC_ALL: es_ES.UTF-8
```

Comment:
honestly love this. want to enable it for roslyn-analyzers once all the tests run on linux